src/lib/conditionals/lightcone/5star/FlameOfBloodBlazeMyPath.ts

```diff
@@ -55,9 +49,22 @@ export default (s: SuperImpositionLevel, withContent: boolean): LightConeConditi
       const r = action.lightConeConditionals as Conditionals<typeof content>
 
       buffAbilityDmg(x, SKILL_DMG_TYPE | ULT_DMG_TYPE, (r.skillUltDmgBoost) ? sValuesSkillUltDmg[s] : 0, SOURCE_LC)
-      buffAbilityDmg(x, SKILL_DMG_TYPE | ULT_DMG_TYPE, (r.skillUltDmgBoost && r.bonusBoost) ? sValuesSkillUltDmg[s] : 0, SOURCE_LC)
     },
-    finalizeCalculations: () => {
+    finalizeCalculations: (x, action, context) => {
+      const r = action.lightConeConditionals as Conditionals<typeof content>
+      if (x.a[Key.HP] * sValuesHpDrain[s] > 500) {
+        buffAbilityDmg(x, SKILL_DMG_TYPE | ULT_DMG_TYPE, (r.skillUltDmgBoost) ? sValuesSkillUltDmg[s] : 0, SOURCE_LC)
+      }
+    },
+    gpuFinalizeCalculations: (action, context) => {
+      const r = action.lightConeConditionals as Conditionals<typeof content>
+      return `
+if (${wgslFalse(r.skillUltDmgBoost)}) {
+  return;
+}
+(*p_x).SKILL_DMG += select(0.0, ${sValuesSkillUltDmg[s]}, x.HP * ${sValuesHpDrain[s]} > 500);
+(*p_x).ULT_DMG += select(0.0, ${sValuesSkillUltDmg[s]}, x.HP * ${sValuesHpDrain[s]} > 500);
```

Comment:
These should be `SKILL_DMG_BOOST` and `ULT_DMG_BOOST` respectively for the DMG % bonus, SKILL_DMG / ULT_DMG are for the actual output damage values

Reply:
An easy way to test the wgsl code is `WEBGPU_DEBUG=true` in the console and select "Keep current relics", then the 3 rows are:

Top - CPU stat sim output
Middle - GPU output
Bottom - CPU worker output

![image](https://github.com/user-attachments/assets/ef3b6292-a8df-4ba4-8fc3-a8fa848a1630)